File: 13-fibonacci/Fibo.cs
// C# program to print the first n fibonacci numbers
// Written by M.V.Harish Kumar on 07-01-2024
using System;

namespace Name
{
    class Fibo
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Fibonacci series printer\n");
            Console.Write("Enter the no. of terms: ");
            int n = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine("The first {n} terms of fibonacci series are:");
            var (a, b) = (0, 1);
            for (int i = 0; i < n; i++)
            {
                Console.WriteLine(a);
                (a, b) = (b, a+b);
            }
        }
    }
}

File: 13-fibonacci/fibo.php
<?php
// PHP program to compute first n fibonacci series
// Written by M.V.Harish Kumar on 18-09-2024

echo "Fibonacci series printer\n\n";
$nterms = intval(readline("Enter the no of terms to print: "));
echo "The first $nterms terms of fibonacci series are:\n";
[$a, $b] = [0, 1];
for ($i = 0; $i < $nterms; $i++) {
	echo $a . "\n";
	[$a, $b] = [$b, $a+$b];
}
